### ISSUE TITLE
Make favourites more easily accesible

### ISSUE THREAD

**Issue by oopmarcos** [yang991178/fluent-reader#214]
It's nice to be able to tag posts as favourite, but there is no way of searching for favourites unless you have the view option set to "All articles" or "Starred only".

- It would be nice if there was an entry in the hamburger menu for starred posts.
- Moreover, I would also argue that starred posts should be discoverable through the search box, even if you have the view option set to "Unread only".

Otherwise, really good work, congrats!

**Comment by yang991178** [comment]
Do you mean adding a menu entry that functions the same as toggling the "starred only" view option? From a design perspective, I would rather avoid adding duplicated functions while saving space on the side bar.

**Comment by oopmarcos** [comment]
Yeah, hadn't though about it that way, it's probably redundant to have it in two places, you're right. However, my other point still stands, wouldn't it be logical to be able to search for starred posts even from the "unread only" view option? After all, if you're searching for a post, it most likely is for something you have already read and starred to be able to locate it easier later.

**Comment by yang991178** [comment]
It would probably be more suitable to view the search function as a filter on the current feed, just like the "unread only" option. Showing articles not in the current view may be confusing for some users, at least in my opinion.

**Comment by oopmarcos** [comment]
From my own experience, I find myself using the search function to search for posts that I have read and starred to access them later, I don't see how one would search for posts that they haven't read yet. Just my two cents though.

**Comment by vosian** [comment]
So this made me think about doing a mock up, I personally like this one, but it'd only really work on list view mode: 
![Fluent Reader](https://user-images.githubusercontent.com/9102033/125204513-bb6f7300-e24b-11eb-8d09-291398bbd58f.png)

While this should work in every view mode: 
![Fluent Reader 2](https://user-images.githubusercontent.com/9102033/125204529-d4782400-e24b-11eb-9eb6-0524db4e7dc2.png)

This way changing the filter is only one click away, rather than having to summon the view menu first.

**Comment by dylankenneally** [comment]
Adding my support for the star/favourites feature to be fleshed out. At the moment, I can add a star but have no means of finding it afterwards - which defeats the purpose of having it, or have I missed something?

My ideal would be:

1. in the menu (left-hand pane)
  - Search
  - All articles
  - Favourites
1. for the favourites *not* to be deleted when Clean Up is used, maybe with a preferences option to change that
